src/main/java/uk/gov/register/views/representations/turtle/RecordTurtleWriter.java

```diff
@@ -31,26 +31,29 @@ public RecordTurtleWriter(javax.inject.Provider<RegisterName> registerNameProvid
     @Override
     protected Model rdfModelFor(RecordView view) {
         Entry entry = view.getEntry();
-        ItemView itemView = view.getItemView();
 
         Model recordModel = ModelFactory.createDefaultModel();
         Model entryModel = new EntryTurtleWriter(registerNameProvider, registerResolver).rdfModelFor(entry, false);
-        Model itemModel = new ItemTurtleWriter(registerNameProvider, registerResolver).rdfModelFor(itemView);
 
         Resource recordResource = recordModel.createResource(recordUri(view.getPrimaryKey()).toString());
         addPropertiesToResource(recordResource, entryModel.getResource(entryUri(Integer.toString(entry.getEntryNumber())).toString()));
-        addPropertiesToResource(recordResource, itemModel.getResource(itemUri(itemView.getItemHash().encode()).toString()));
 
         Map<String, String> prefixes = entryModel.getNsPrefixMap();
-        prefixes.putAll(itemModel.getNsPrefixMap());
+
+        view.getItemViews().forEach(iv -> {
```

Comment:
Yes - I think the TTL representations are wrong. I'm going to fix it and add some tests for the Turtle writers. (After I've learned Turtle!)